source/administration/console/subnet-registration.rst

```diff
@@ -0,0 +1,74 @@
+.. _minio-console-subscription:
+
+===================
+SUBNET Registration
```

Comment:
Since we have Support under this, maybe "SUBNET Registration and Support" ?

Reply:
Sure. They are kind of one and the same, which is why I didn't specifically mention it.